examples/nodejs/eventbridge/webapp/README.md

```diff
@@ -0,0 +1,30 @@
+# React + TypeScript + Vite
```

Comment:
we might want to get rid of this file, or update it to represent what our app does.